test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,41 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+        // empty list
+        assertDoesNotContainNull();
+
+        // only one object
+        assertContainNull((Object) null);
+        assertDoesNotContainNull(1);
+        assertDoesNotContainNull("");
+        assertDoesNotContainNull("abc");
+
+        // multiple object
+        assertContainNull((Object)null, "abc");
+        assertContainNull((Object)null, 123);
+        assertContainNull((Object)null, "");
+        assertContainNull((Object)null, (Object)null);
+        assertDoesNotContainNull("" , "");
```

Comment:
Some of these test cases may be repetitive since it is already tested in the "only one object" test cases.
e.g. Since "" is already, you can switch it out and have other test cases testing more than 2 objects